src/fides/api/models/privacy_request.py

```diff
@@ -950,6 +951,9 @@ def error_processing(self, db: Session) -> None:
             db=db, data={"message_sent": False, "privacy_request_id": self.id}
         )
 
+    def get_log_context(self):
+        return {"privacy_request_id": self.id}
```

Comment:
nit: the key here should be the new enum you've defined (in the other PR), right?